governance/polls/Proposal - Adjust the Dust Parameter

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Please ensure the filename has the .md suffix.

Reply:
👍